src/backend/dist.js

```diff
@@ -1,204 +1,273 @@
+/* eslint-disable no-empty-label */
 'use strict';
 
 const fs = require('fs-extra');
-var logger = require('./buildlogger.js');
-var zip = require('decompress-zip');
+const logger = require('./buildlogger.js');
+const zip = require('decompress-zip');
 const config = require('../../config.json');
 const request = require('request').defaults({'proxy': config.proxy});
 const async = require('async');
 const progressStream = require('progress-stream');
 const streamBuffer = require('stream-buffers');
-const path = require("path");
+const path = require('path');
 const recursive = require('recursive-readdir');
 const sharp = require('sharp');
 
 const distPath = __dirname + '/../../dist';
 const uploadsPath = __dirname + '/../../uploads';
 const mockPath = __dirname + '/../../mockjson';
+let width, height, ratio, padding, diffHeight, qualityMultiplier, filePath, fileData, hashString, counter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variables within the same declaration block should be sorted alphabetically. (sort-vars)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)